functorch/csrc/dim/minpybind.h

```diff
@@ -385,10 +385,6 @@ bool is_int(handle h) {
     return PyLong_Check(h.ptr());
 }
 
-bool is_float(handle h) {
```

Comment:
This is a library / a minified lightweight version of pybind11 and this could be useful later.

Reply:
It is easy to be added back later.